### PR TITLE
Nv 1476 web   get layouts endpoint

### PR DESCRIPTION
**Pull request by ainouzgali**
### Why was this change needed?

Get list of layouts

### CODE REVIEW

File: libs/shared/src/entities/layout/layout.interface.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  name: LayoutName;
+  channel: ChannelTypeEnum;
+  content: string;
+  contentType: string;
+  variables?: ITemplateVariable[];
+  isDefault: boolean;
+  isDeleted: boolean;
+  createdAt?: string;
+  updatedAt?: string;
+}
```
Comment (p-fernandez) [inline libs/shared/src/entities/layout/layout.interface.ts:18]: Let's make this shared and call it DTO (data transfer object) as entity we are using it more for the Repository objects. What do you think? Also like this we will have it common for both.
Unless if for your needs you prefer to have a separate object but might be more difficult to keep in sync with backend changes. Up to your preference. 🙂

**Review by p-fernandez — APPROVED** [review]
🌟

File: libs/shared/src/entities/layout/layout.interface.ts (hunk 1 above)
Comment (ainouzgali) [reply libs/shared/src/entities/layout/layout.interface.ts:18]: I actually did that as a dto initially. But then felt its odd to use dto in the web. I am actually not sure what is our practice here. We seem to not have consistency about that. 
I'll do a bit more research here

Comment (ainouzgali) [reply libs/shared/src/entities/layout/layout.interface.ts:18]: @davidsoderberg wdyt?

Comment (ainouzgali) [reply libs/shared/src/entities/layout/layout.interface.ts:18]: I am merging this for now

**Pull request merged**